From 1ea8f9817db8eba9e8042b44062df2112984e4e5 Mon Sep 17 00:00:00 2001
From: Roland Haas <rhaas@tapir.caltech.edu>
Date: Fri, 31 Jan 2014 10:32:14 -0800
Subject: [PATCH 31/36] GRHydro: fix reconstruct_Wv in F90 PPMM routine

was using velx in some places where vx should be used
---
 src/GRHydro_PPMM.F90 | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/src/GRHydro_PPMM.F90 b/src/GRHydro_PPMM.F90
index 63a8e2f..82f95bc 100644
--- a/src/GRHydro_PPMM.F90
+++ b/src/GRHydro_PPMM.F90
@@ -137,9 +137,9 @@ trivial_rp = .true.
       dmx(i) = 0.d0                                                &&\
     end if
     STEEP(rho, drho, dmrho)
-    STEEP(velx, dvelx, dmvelx)
-    STEEP(vely, dvely, dmvely)
-    STEEP(velz, dvelz, dmvelz)
+    STEEP(vx, dvelx, dmvelx)
+    STEEP(vy, dvely, dmvely)
+    STEEP(vz, dvelz, dmvelz)
     STEEP(Bvcx, dBvcx, dmBvcx)
     STEEP(Bvcy, dBvcy, dmBvcy)
     STEEP(Bvcz, dBvcz, dmBvcz)
@@ -362,7 +362,7 @@ trivial_rp = .true.
 
   do i = 3, nx - 2
     dpress2 = press(i+2) - press(i-2)
-    dvel = velx(i+1) - velx(i-1)
+    dvel = vx(i+1) - vx(i-1)
     if ( (abs(dpress(i)) >  ppm_epsilon * min(press(i-1),press(i+1))) .and. &
          (dvel < 0.d0) ) then
       w = 1.d0
@@ -472,9 +472,9 @@ do i = GRHydro_stencil, nx - GRHydro_stencil + 1
     end if
 
     MON(rhominus,rho,rhoplus)
-    MON(velxminus,velx,velxplus)
-    MON(velyminus,vely,velyplus)
-    MON(velzminus,velz,velzplus)
+    MON(velxminus,vx,velxplus)
+    MON(velyminus,vy,velyplus)
+    MON(velzminus,vz,velzplus)
     MON(Bvcxminus,Bvcx,Bvcxplus)
     MON(Bvcyminus,Bvcy,Bvcyplus)
     MON(Bvczminus,Bvcz,Bvczplus)
-- 
1.8.5.3

